### PR TITLE
CCT-2088: Paypal Commerce plugin documentation

### PR DESCRIPTION
**Pull request by zzzarka**
https://centracommerce.atlassian.net/browse/CCT-2088

### CODE REVIEW

**Review by sirtopp — CHANGES_REQUESTED** [review]
Tiny feedback. Docs are understandable for an average dev to start using it. Small editorial issues.

File: source/guides/plugins/paypal-commerce.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+This plugin enables payments with a customers paypal account or with creditcard.
+
+## Set up
+
+To configure Paypal Commerce plugin in Centra go to store plugins, select desirable plugin and you should see similar screen to the screenshot below.
+
+```eval_rst
+.. image:: images/paypal-commerce-setup.png
+   :scale: 30 %
```
[the remark this answers is not in the document]
Comment (zzzarka) [reply source/guides/plugins/paypal-commerce.md:11]: no idea :D i copied this from Stripe docs :) but i agree it is tiny so i will scale it up.

File: source/guides/plugins/paypal-commerce.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+    const billingAddress: IAddress = Address.create(billingAddressData);
+    const shippingAddress: IAddress = Address.create(shippingAddressData);
+    checkout({
+      paymentMethodSpecificFields,
+      paymentMethod: paymentMethod,
+      billingAddress: billingAddress,
+      shippingAddress: shippingAddress,
+    });
+  }
+```
```
[the remark this answers is not in the document]
Comment (zzzarka) [reply source/guides/plugins/paypal-commerce.md:136]: :D this was copied from stripe .. or maybe adyen documentation .. i will see if that has something to say about it :)

File: source/guides/plugins/paypal-commerce.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+
+## Limitations
+
+We require the customer to select the correct country and state before starting the paypal checkout process.
+
+## Usage
+
+This plugin can be used in 2 ways either the checkout redirects to paypal or you render what payapl calls msart buttons on the checkout page which enables payments with either credit card or paypal account. 
+
+## Redirect
```
[the remark this answers is not in the document]
Comment (sirtopp) [reply source/guides/plugins/paypal-commerce.md:29]: Yup, makes sense. :)

File: source/guides/plugins/paypal-commerce.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (sirtopp) [reply source/guides/plugins/paypal-commerce.md:147]: Ok, so it stays that way then. :)

**Review by sirtopp — APPROVED** [review]

**Pull request closed without merging**